manifests/y/Yandex/Chats/YandexMessenger/2.170.0/Yandex.Chats.YandexMessenger.locale.ru.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.5.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: Yandex.Chats.YandexMessenger
+PackageVersion: 2.170.0
+PackageLocale: ru
+Publisher: Yandex.Chats
+PackageName: Yandex Messenger 2.170.0

```

```suggestion
PackageName: Yandex Messenger
```

Fuzzy matching is used for PackageName. We can omit the version information that changes for every new release and it wont affect the package matching.